src/test/java/dk/sdu/se_f22/contentmodule/stopwords/SemesterProjekt1.4_stopWordFilter/src/StopWords.java

```diff
@@ -0,0 +1,161 @@
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Scanner;
+
+public class StopWords implements IStopWords {

```

Comment:
Wouldn't a singleton be beneficial here? Since currently if 3 classes rely on the StopWords filter, they all need to instantiate a StopWords class, and therefore spin up 3 db connections.

Perhaps even use the shared DB connection?